x/supplier/config/supplier_configs_reader_test.go

```diff
@@ -360,15 +484,64 @@ func Test_ParseSupplierConfigs_Services(t *testing.T) {
 				      rpc_type: json_rpc
 				      config:
 				        timeout: 10
-				`,
+				`, ownerAddress, operatorAddress),
 			expectedError: config.ErrSupplierConfigInvalidStake,
 		},
+		{
+			desc: "missing owner address",
+			inputConfig: fmt.Sprintf(`
+				# explictly omitted owner address
+				operator_address: %s
+				stake_amount: 1000upokt
+				services:
+				  - service_id: svc
+				    endpoints:
+				    - publicly_exposed_url: http://pokt.network:8081
+				      rpc_type: json_rpc
+				      config:
+				        timeout: 10
+				`, operatorAddress),
+			expectedError: config.ErrSupplierConfigInvalidAddress,
+		},
+		{
+			desc: "invalid owner address",
+			inputConfig: fmt.Sprintf(`
+				owner_address: invalid_address
+				operator_address: %s
+				stake_amount: 1000upokt
+				services:
+				  - service_id: svc
+				    endpoints:
+				    - publicly_exposed_url: http://pokt.network:8081
+				      rpc_type: json_rpc
+				      config:
+				        timeout: 10
+				`, operatorAddress),
+			expectedError: config.ErrSupplierConfigInvalidAddress,
+		},
+		{
+			desc: "invalid operator address",
```

Comment:
Is there one when operator_address is explicitly ommited for a success scenario?

Reply:
Added